tests/transforms/test_qcut.py

```diff
@@ -2003,6 +2071,25 @@ def circuit(x):
 
         assert np.isclose(gradient, cut_gradient)
 
+    def test_circuit_with_disconnected_components(self, use_opt_einsum, mocker):
+        """Tests if a circuit that is fragmented into subcircuits such that some of the subcircuits
+        are disconnected from the final terminal measurements is executed correctly"""
+        dev = qml.device("default.qubit", wires=1)
```

Comment:
cool to see the device wires is smaller than required for full circuit :sunglasses: